### PR TITLE
[kodi] New features: Audio and Subtitle and IMDB ID

### PR DESCRIPTION
**Pull request by m17design**
New features:

Some media have different stream like English/Japanese/Chinese or commentary audio or CC subtitle, Kodi can play and select audio or subtitle from this media file. New features add support to select and display language infomation.

1: Enable or disable subtitle.
2: Select subtitle and audio from multi stream.
3: Language of current audio or subtitle.

Media link added more infomation for create infomation page and link for current media. User can using these infomation or id to generate media page and link by using Rule or RESTAPI:

4: Imdb id, Tmdb id, Douban id and media_id from database of kodi .
5: Width and height of current video.
6: Channels number of current audio.

### CODE REVIEW

**Comment by davidgraeff** [comment]
Can you please have a look at the conflicts? :)

File: addons/binding/org.openhab.binding.meteoblue/README.md [hunk 1]
```diff
@@ -123,19 +123,18 @@ Group weatherDay6 "Weather in 6 days"
...
-String todayIcon     "Icon [%s]"       (weatherDay0)  {channel="meteoblue:weather:metBridge:A51:forecastToday#icon"}
+String todayCond     "Condition [%s]"  (weatherDay0)  {channel="meteoblue:weather:metBridge:A51:forecastToday#condition"}
+Image todayIcon     "Icon"       (weatherDay0)  {channel="meteoblue:weather:metBridge:A51:forecastToday#icon"}
 Number todayUV       "UV Index [%d]"  (weatherDay0)  {channel="meteoblue:weather:metBridge:A51:forecastToday#UVIndex"}
-Number:Temperature  todayTempL  "Low Temp [%.2f �F]"   <temperature>  (weatherDay0)  {channel="meteoblue:weather:metBridge:A51:forecastToday#minTemperature"}
-Number:Temperature  todayTempH  "High Temp [%.2f �F]"  <temperature>  (weatherDay0)  {channel="meteoblue:weather:metBridge:A51:forecastToday#maxTemperature"}
+Number:Temperature  todayTempL  "Low Temp [%.2f °F]"   <temperature>  (weatherDay0)  {channel="meteoblue:weather:metBridge:A51:forecastToday#minTemperature"}
+Number:Temperature  todayTempH  "High Temp [%.2f °F]"  <temperature>  (weatherDay0)  {channel="meteoblue:weather:metBridge:A51:forecastToday#maxTemperature"}
```
Comment (wborn) [inline addons/binding/org.openhab.binding.meteoblue/README.md]: Please remove these Meteoblue changes from the PR.

Comment (m17design) [reply addons/binding/org.openhab.binding.meteoblue/README.md]: I'm a newbie of github, I don't know why this commit include in my PR. I don't know how to remove it from my PR.  Can you help me learning how to do that.

Comment (wborn) [reply addons/binding/org.openhab.binding.meteoblue/README.md]: Just add a new commit that modifies the file so the file has the content of the current master again:

https://github.com/openhab/openhab2-addons/blob/master/bundles/org.openhab.binding.meteoblue/README.md

**Comment by Hilbrand** [comment]
You still seem to have commits in here that should not be part of this pull request. You can remove them with `git rebase -i HEAD~<number of commits to go back>`. I didn't check what the number should be, but it should be the first commit showing is initial commit for Spotify and than remove till the meteoblue commit. Than push force back.

You must also sign off with your real name, not you github account. See https://www.openhab.org/docs/developer/contributing/contributing.html#sign-your-work
(While you're rebasing using the above command you could squash your own commits easing correcting the sign off).

**Comment by m17design** [comment]
Sorry. I can't figure out how to fix wrong commits seems I work on an old branch. I have made "org.openhab.binding.kodi-2.5.0-SNAPSHOT.jar" in my computer with eclipse. I will commits again when I know how to use git.

**Pull request closed without merging**